operator/src/main/java/org/keycloak/operator/OperatorMain.java

```diff
@@ -0,0 +1,25 @@
+package org.keycloak.operator;
+
+import io.javaoperatorsdk.operator.Operator;
+import io.quarkus.runtime.Quarkus;
+import io.quarkus.runtime.QuarkusApplication;
+import io.quarkus.runtime.annotations.QuarkusMain;
+
+import javax.inject.Inject;
+
+@QuarkusMain
+public class OperatorMain implements QuarkusApplication {
```

Comment:
Is this `Main` needed? What's the reason fo having it?

Reply:
To not have an Operator+Informers instance already created and started by the DI , during the tests.

Reply:
We are not using `QuarkusMain` tests, I'm really surprised that this is the behaviour, do you have a link to an open issue? Have you attempted to run the tests now without this Main, what's the error that you get?

Reply:
Last time I tried I had issues with the informers as they were registered to the default namespace ....

Reply:
Running with `remote` and without this `Main` works on my machine.

Unfortunately with `local` I'm reliably getting a NPE (with and without the `Main`):
```
[ERROR] Errors: 
[ERROR]   OperatorE2EIT>ClusterOperatorTest.before:61 » NullPointer
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

Reply:
Are you running locally like this : 
`mvn clean verify \
              -Dquarkus.kubernetes.deployment-target=kubernetes \
              --no-transfer-progress -Dtest.operator.deployment=local
`

Reply:
tests are running in `local` and `remote` mode for me too now 👍